docs/community/governance.md

```diff
@@ -1,76 +1,84 @@
 
-- Team members are added to the [GitHub project](https://github.com/grafana/loki) as Collaborator (Push access to the repository).
-- **Team member must enable [two-factor authentication](https://help.github.com/articles/about-two-factor-authentication) on their GitHub account**
-- Team members are added to the [team mailing list](mailto:loki-team@googlegroups.com).
-- Team members are added to the list of team members in this document.
-- New team members are announced on the [discussion group](https://groups.google.com/forum/#!forum/lokiproject) and Loki [slack channel](https://grafana.slack.com/messages/CEPJRLQNL) by an existing team member.
+Changes in maintainership have to be announced on the [developers mailing list][devs]. They are decided by [rough consensus](#consensus) and formalized by changing the [`MAINTAINERS.md`][maintainers] file of the respective repository.
 
-Team members may retire at any time by emailing the [team](mailto:loki-team@googlegroups.com).
+Maintainers are granted commit rights to all projects covered by this governance.
 
-Team members can be removed by [supermajority vote](#supermajority-vote) on the team [mailing list](mailto:loki-team@googlegroups.com). For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+A maintainer or committer may resign by notifying the [team mailing list][team]. A maintainer with no project activity for a year is considered to have resigned. Maintainers that wish to resign are encouraged to propose another team member to take over the project.
 
-Upon death of a member, their team membership ends automatically.
+A project may have multiple maintainers, as long as the responsibilities are clearly agreed upon between them. This includes coordinating who handles which issues and pull requests.
 
 ### Technical decisions
 
-Technical decisions are made informally by the team member, and [lazy consensus](#consensus) is assumed. If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
+Technical decisions that only affect a single project are made informally by the maintainer of this project, and [rough consensus](#consensus) is assumed. Technical decisions that span multiple parts of the project should be discussed and made on the [developer mailing list][devs].
+
+Decisions are usually made by [rough consensus](#consensus). If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
 
 ### Governance changes
 
-Material changes to this document are discussed publicly on as an issue on the Loki repository. Any change requires a [supermajority](#supermajority-vote) in favor. Editorial changes may be made by [lazy consensus](#consensus) unless challenged.
+Changes to this document are made by Grafana Labs.
```

Comment:
should be a bit more specific?

Changes can be made to this document at any time without a formal vote by Grafana Labs

Should it also include changes to maintainers/team-members?